web/web-portlet/src/bundle/vm/edit.vm

```diff
@@ -5,6 +5,12 @@ ${includeLatestJQuery}
 <script>
 	includeLatestJQuery('web-ifp edit.vm');
 	includeWebjarLibrary("fontawesome-iconpicker");
+	//When hiding the iconpicker focus back on the element
+	$( document ).ready(function() {
+	    jQuery("#fa_icon").on("iconpickerHide", function (event) {
```

Comment:
jQuery or $ :), pick one!

Reply:
Oops! ;)

Reply:
I also wanted to try to get the focus of the next tabindex'able object but I wasn't sure how to do that without directly getting the id, which is probably fine in this case. Putting the focus back to where it was seemed fine though and better than it was. I'm not sure why this is working as expected in LTI, something with the HTML layout?